src/pages/en/reference/integrations-reference.md

```diff
@@ -194,15 +194,20 @@ A read-only copy of the user-supplied [Astro config](/en/reference/configuration
 A mutable instance of the Vite server used in "dev" and "preview" mode. For instance, this is [used by our Partytown integration](https://github.com/withastro/astro/tree/main/packages/integrations/partytown) to inject the Partytown server as middleware:
 
 ```js
-import
-
-'astro:server:setup': ({ server }) => {
-  server.middlewares.use(
-    partytownServer(partytownLibDirectory, {
-      mount: '/~partytown',
-      ...
-    })
-  );
+import partytown from '@astrojs/partytown'
+
+export default {
+  name: 'partytown'
+  hooks: {
+    'astro:server:setup': ({ server }) => {
+      server.middlewares.use(
+        partytownServer(partytownLibDirectory, {
+          mount: '/~partytown',
+          // ...
+        })
```

Comment:
I think we can make this more generic. This suggestion is based on [the signature used inside the partytown integration](https://github.com/withastro/astro/blob/08432d5b01a5cab1bd8d962c30f77ab3827bc5aa/packages/integrations/partytown/src/sirv.ts#L205).

```suggestion
        function middleware(req, res, next) {
          // handle requests
        }
```